src/main/resources/org/sagebionetworks/web/client/widget/evaluation/EvaluationRowWidget.ui.xml

```diff
@@ -100,7 +100,7 @@
             addStyleNames="displayInline"
           />
         </b:FormGroup>
-        <bh:Div ui:field="quotaUI" addStyleNames="light-border">
+        <bh:Div ui:field="quotaUI">
```

Comment:
I have not been able to visually inspect this, as this UI is only shown for old challenges that use the "quota" system